server/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const app = express();
+const parser = require('body-parser');
+const path = require('path');
```

Comment:
Not used.

Reply:
Path will be used once I get whatever bizzaro thing that is happening with my static files fixed.